document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/AccountsHandlerImpl.java

```diff
@@ -32,17 +36,27 @@ public DocumentInfoResponse getAccountsData(String resourceLink) throws HandlerE
             throw new HandlerException(e.getMessage(), e.getCause());
         }
 
-        // TODO: The accountsType variable shall be used in the implementation of abridged logic
         AccountType accountsType = getAccountType(accounts);
 
+        String abridgedAccountLink = getAccountLink(accounts, accountsType);
```

Comment:
The account type is being retrieved from the links to then retrieve the link using the account type. Is the account type going to be used for anything else or can this be simplified to just get the non-self link?